libs/ui-toolkit/button/src/lib/button.component.scss

```diff
@@ -32,9 +32,9 @@ button:disabled {
 }
 
 .green {
```

Comment:
I know the idea is to not break previous configuration and avoid refactors, but it's very weird to use `btnType="green"` and have a `.green` class when that doesn't represent the actual color of the button...

How about change this to `.primary`, change `.gray` to `.secondary`, and then add a `set` function in the `btnType` input that makes the conversion from the old `green` and `gray` styles to the new ones?

Reply:
Agree, we'll do this later. We've some "base-button" lost in some part of our system, the idea is to eventually merge both implementation